apps/journeys-admin/src/components/Team/CopyToTeamMenuItem/CopyToTeamMenuItem.spec.tsx

```
@@ -119,6 +118,75 @@
+          id: 'translatedJourneyId',
+          title: 'Viaje Traducido',
+          description: 'Esta es una descripción traducida',
+          languageId: 'es-419',
+          createdAt: '2023-04-25T12:34:56Z',
+          updatedAt: '2023-04-25T12:34:56Z'
+        }
+      }
+    }
+  }
+
+  const duplicateJourneyMock: MockedResponse<JourneyDuplicate> = {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'duplicateJourneyMock'. Consider to delete it or rename it.

'duplicateJourneyMock' is defined here:


(lint/suspicious/noRedeclare)